specs/ClearBrowsingData.md

```diff
@@ -0,0 +1,268 @@
+    if (webView.CoreWebView2 != null) 
+    {
+        profile = webView.CoreWebView2.Profile;
+        double endTime = DateTimeOffset.Now.ToUnixTimeSeconds();
+        // Get the current time which is the time in which the browsing data will be cleared until.
+        double startTime = endTime - 3600;
+        // Offset the current time by 3600 seconds to clear the browsing data from the last hour.
+        CoreWebView2BrowsingDataKinds dataKinds = (CoreWebView2BrowsingDataKinds)(CoreWebView2BrowsingDataKinds.GeneralAutofill | CoreWebView2BrowsingDataKinds.PasswordAutosave);
+        await profile.ClearBrowsingDataAsync(dataKinds, startTime, endTime);
+    }
+}
+
+```
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+
+# API Details
+
+## Win32 C++
+
+```IDL
+interface ICoreWebView2ClearBrowsingDataCompletedHandler;
+
+/// Specifies the datatype for the
+/// `ICoreWebView2Profile::ClearBrowsingData` method.
+[v1_enum]
+typedef enum COREWEBVIEW2_BROWSING_DATA_KINDS {
+  /// Specifies data stored by the AppCache DOM feature.
+  COREWEBVIEW2_BROWSING_DATA_KINDS_APP_CACHE = 1 << 0,
+
+  /// Specifies file systems data.
+  COREWEBVIEW2_BROWSING_DATA_KINDS_FILE_SYSTEMS = 1 << 1,
+
+  /// Specifies data stored by the IndexedDB DOM feature.
+  COREWEBVIEW2_BROWSING_DATA_KINDS_INDEXED_DB = 1 << 2,
+
+  /// Specifies data stored by the localStorage DOM API.
+  COREWEBVIEW2_BROWSING_DATA_KINDS_LOCAL_STORAGE = 1 << 3,
+
+  /// Specifies data stored by the Web SQL database DOM API.
+  COREWEBVIEW2_BROWSING_DATA_KINDS_WEB_SQL = 1 << 4,
+
+  /// Specifies cache storage which stores the network requests
+  /// and responses. 
+  COREWEBVIEW2_BROWSING_DATA_KINDS_CACHE_STORAGE = 1 << 5,
+
+  /// Specifies DOM storage data. This browsing data kind is inclusive 
+  /// of COREWEBVIEW2_BROWSING_DATA_KINDS_APP_CACHE, 
+  /// COREWEBVIEW2_BROWSING_DATA_KINDS_FILE_SYSTEMS,
+  /// COREWEBVIEW2_BROWSING_DATA_KINDS_INDEXED_DB, 
+  /// COREWEBVIEW2_BROWSING_DATA_KINDS_LOCAL_STORAGE,
+  /// COREWEBVIEW2_BROWSING_DATA_KINDS_WEB_SQL, 
+  /// COREWEBVIEW2_BROWSING_DATA_KINDS_CACHE_STORAGE.
+  COREWEBVIEW2_BROWSING_DATA_KINDS_DOM_STORAGE = 
+      COREWEBVIEW2_BROWSING_DATA_KINDS_APP_CACHE|
+      COREWEBVIEW2_BROWSING_DATA_KINDS_FILE_SYSTEMS | 
+      COREWEBVIEW2_BROWSING_DATA_KINDS_INDEXED_DB |
+      COREWEBVIEW2_BROWSING_DATA_KINDS_LOCAL_STORAGE | 
+      COREWEBVIEW2_BROWSING_DATA_KINDS_WEB_SQL |
+      COREWEBVIEW2_BROWSING_DATA_KINDS_CACHE_STORAGE,
+
+  /// Specifies HTTP cookies data.
+  COREWEBVIEW2_BROWSING_DATA_KINDS_COOKIES = 1 << 6,
+
+  /// Specifies site data. This browsing data kind
+  /// is inclusive of COREWEBVIEW2_BROWSING_DATA_KINDS_DOM_STORAGE and
+  /// COREWEBVIEW2_BROWSING_DATA_KINDS_COOKIES. 
+  COREWEBVIEW2_BROWSING_DATA_KINDS_SITE = COREWEBVIEW2_BROWSING_DATA_KINDS_DOM_STORAGE | 
+      COREWEBVIEW2_BROWSING_DATA_KINDS_COOKIES, 
+
+  /// Specifies content in the HTTP cache including images and other files. 
```

Comment:
How is this different from "cache storage which stores the network requests and responses"?

What about the HTTPS cache?

Reply:
Great question! Please update documentation to make it clearer what these different enum values are covering.